.gitignore

```diff
@@ -53,5 +53,6 @@ terraform.tfstate.d/
 lock.json
 **/.terraform/
 terraform/**/*.yml
+terraform/**/*.json
```

Comment:
These shouldn't be required anymore:
```
terraform/**/*.yml
terraform/**/*.json
terraform/**/*.base64
!terraform/workspace_variables/app_config.yml
```

Reply:
I agree.... but could leave until after this PR.

Reply:
all done....